### PR TITLE
Documentation/op-guide: Clarify TLS "--peer" flags

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/9164.

### CODE REVIEW

File: Documentation/op-guide/configuration.md [hunk 1]
```diff
@@ -271,12 +271,12 @@ The security flags help to [build a secure etcd cluster][security].
 + env variable: ETCD_PEER_CA_FILE
 
 ### --peer-cert-file
-+ Path to the peer server TLS cert file.
++ Path to the peer server TLS cert file. This is the cert for peer-to-peer traffic, used both for transport and listener.
```
Comment (xiang90) [inline Documentation/op-guide/configuration.md]: used both for server and client.

**Comment by ericchiang** [comment]
lgtm

**Comment by rithujohn191** [comment]
@gyuho thank you for this

**Pull request merged**